sanitized_dump/utils/models.py

```diff
@@ -36,3 +36,26 @@ def get_model_field_names(model):
 def get_model_table_names():
     models = get_models()
     return [get_model_table_name(model) for model in models]
+
+
+def get_model_table_map():
+    models = get_models()
+    return {get_model_table_name(model): model for model in models}
+
+
+def validate_all_model_fields_in_config(config):
+    strategy = config.get('strategy', None)
+    if not strategy:
+        return False
+
+    model_table_map = get_model_table_map()
+    model_table_names = model_table_map.keys()
+    for model_table_name, fields in strategy.items():
+        if model_table_name not in model_table_names:
+            return False
+        model = model_table_map[model_table_name]
+        model_field_names = get_model_field_names(model)
+        if set(fields.keys()) != set(model_field_names):
+            return False
```

Comment:
I think this could take the diff and return list of `<ModelName>.<FieldName>` strings that weren't present in the conf. That way you could then loop the list and print something nice to the command line.

We can implement that in following PRs.